subsys/esb/esb.c

```diff
@@ -1436,8 +1440,9 @@ int esb_init(const struct esb_config *config)
 
 	disable_event.event.generic.event = esb_ppi_radio_disabled_get();
 
-	nrf_radio_modecnf0_set(NRF_RADIO, esb_cfg.use_fast_ramp_up,
-		nrf_radio_modecnf0_dtx_get(NRF_RADIO));
+#if CONFIG_ESB_FAST_RAMP_UP
+	nrf_radio_modecnf0_set(NRF_RADIO, true, nrf_radio_modecnf0_dtx_get(NRF_RADIO));
+#endif
```

Comment:
Is it okay that now this function is not called when the fast ramp up is not enabled?
Perhaps it would be better to use:
```
	nrf_radio_modecnf0_set(NRF_RADIO, IS_ENABLED(CONFIG_ESB_FAST_RAMP_UP),
		nrf_radio_modecnf0_dtx_get(NRF_RADIO));
```